packages/fxa-payments-server/src/branding.ftl

```diff
@@ -21,7 +21,9 @@
 
 # “Accounts” can be localized, “Firefox” must be treated as a brand.
 # 'Firefox Accounts' refers to the service
+# TODO - Remove once branding sticks
 project-brand = Firefox Accounts
+project-brand-2 = { - brand-name-mozilla } account
```

Comment:
We should use the brand term for this.

If we're removing this file in the future, but still plan to use this string, it might be better to place this in a different resource file somewhere. I'd also update the ID and comment to be more descriptive - especially since some languages will need to determine what sort of capitalization they should use.

Note - the ID and comment are both suggestions, please feel free to improve on them.

Reply:
Thanks for the call out. I have updated this accordingly. Your comment and id seem great, so I used them.